pre-docs/inventory_version2.md

```diff
@@ -76,6 +76,93 @@ plan setup_lb (
 }
 ```
 
+## Creating targets in plans
+
+When using inventory v2 a new and improved API for interacting with `Target`s in inventory is available. Two new plan functions have been added: `get_target` which allows retrieving a single `Target` from inventory and `set_config` which allows setting config on a specific `Target`. The updated API also provides a way to instantiate new `Target`s with data that more closely resembles how targets are declared in an inventory file.
```

Comment:
I'm not sure if Chris will want this to change, but I like that `Target` is caps and preformatted, since it indicates that it's an object.

Reply:
@puppetlabs/bolt Open question

Reply:
Oh I think we can just ping @cwcowellshah on this - was just stating that the formatting may seem weird, but I like it as it is.